### PR TITLE
fix(CX-1759): change all titles and subtitles from Sentence Case to Title Case in Eigen

### PR DESCRIPTION
**Pull request by Nargle30**
The type of this PR is: **Enhancement**

This PR resolves [CX-1759]

### Description

In this PR  all titles and subtitles of SectionTitle component were changed to Title Case. Library @artsy/to-title-case was used to do so. 
Almost all of the titles and subtitles have been already formatted correctly, so there should not be a lot visible changes in the app. Titles and subtitles that come from metaphysics are fixed in [this PR](https://github.com/artsy/metaphysics/pull/2610)

https://user-images.githubusercontent.com/36167539/148576404-db973c6c-c7aa-42b5-ae98-f7ce7645de9a.mp4

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/main/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/main/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Changed titles and subtitles of SectionTitle component to Title Case -daria

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

[CX-1759]: https://artsyproduct.atlassian.net/browse/CX-1759?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (Changed titles and subtitles of SectionTitle component to Title Case -daria)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 142fce97c9d473d17115a7c96731efd6f1a9c25c
</p>

**Review by olerichter00 — COMMENTED** [review]
Nice, I like the approach a lot! 

I wonder if the subtitle needs to be updated as well. I've commented on the Jira ticket to clarify this question. Feel free to merge once this is resolved 🚀

**Comment by Nargle30** [comment]
> Nice, I like the approach a lot!
> 
> I wonder if the subtitle needs to be updated as well. I've commented on the Jira ticket to clarify this question. Feel free to merge once this is resolved 🚀

Thanks :)
Oki, i'll merge

**Comment by olerichter00** [comment]
> > Nice, I like the approach a lot!
> > I wonder if the subtitle needs to be updated as well. I've commented on the Jira ticket to clarify this question. Feel free to merge once this is resolved 🚀
> 
> Thanks :) Oki, i'll merge

Ok,

File: src/lib/Components/SectionTitle.tsx [hunk 1]
```diff
@@ -27,11 +28,11 @@ export const SectionTitle: React.FC<{
       <Flex mb={2} flexDirection="row" alignItems="center">
         <View style={{ overflow: "hidden", flex: 1 }}>
           <Text lineHeight="20" variant="sm" ellipsizeMode="tail" numberOfLines={1} testID="title">
-            {title}
+            {toTitleCase(String(title))}
           </Text>
           {Boolean(subtitle) && (
             <Text variant="sm" color="black60" lineHeight="20" testID="subtitle">
-              {subtitle}
+              {toTitleCase(String(subtitle))}
```
Comment (olerichter00) [inline src/lib/Components/SectionTitle.tsx:35]: As far as I understand the comment on the Jira ticket the subtitle doesn't need to be titleCased. I guess, the title of the Jira ticket was just wrong. Sorry for the confusion 🙇‍♂️

**Review by olerichter00 — CHANGES_REQUESTED** [review]

**Review by olerichter00 — APPROVED** [review]

**Review by olerichter00 — APPROVED** [review]
Feel free to merge after updating the failing tests.

**Pull request merged**